server/src-lib/Hasura/GraphQL/Transport/WebSocket.hs

```diff
@@ -603,8 +612,9 @@ createWSServerApp
   -> WSServerEnv
   -> WS.PendingConnection -> m ()
   -- ^ aka generalized 'WS.ServerApp'
-createWSServerApp authMode serverEnv =
-  WS.createServerApp (_wseServer serverEnv) handlers
+{-# INLINABLE createWSServerApp #-}
```

Comment:
This `INLINABLE` pragma sadly will not do anything because the `HasVersion` implicit parameter defeats specialization. Is there a reason we really care about this being specialized?

Reply:
Thanks, That was leftover from an experiment I abandoned, just removed it in latest push.